src/Compilers/CSharp/Portable/BoundTree/Formatting.cs

```diff
@@ -120,6 +120,14 @@ public override object Display
         }
     }
 
+    internal partial class BoundDiscardedExpression
+    {
+        public override object Display
+        {
+            get { return string.Empty; }
```

Comment:
The only thing I needed was for `Display` not to crash. The returned value is not observable currently, even in overload errors.